tests/test_bgp_speaker.py

```diff
@@ -65,7 +65,7 @@ def common_setup_teardown(duthost, ptfhost):
         duthost.command("ip route flush %s/32" % ip.ip)
         duthost.command("ip route add %s/32 dev %s" % (ip.ip, mg_facts['minigraph_vlan_interfaces'][0]['attachto']))
 
-    port_num = [5000, 6000, 7000]
+    port_num = [7000, 8000, 9000]
```

Comment:
How can we know the used port numbers? any reference?

Reply:
Here is a reference:

https://github.com/tahmed-dev/sonic-mgmt/blob/taahme/fix-exabgp-ports-in-bgp-speaker-test/tests/fib.py#L107

Reply:
It's difficult to keep track of which port numbers are used by each test. There is a good possibility we will encounter conflicts like this in the future. We might want to consider loading port numbers and other data from some sort of config file in the future.